ShadowedUnitFrames.lua

```diff
@@ -698,9 +698,11 @@ local function hideBlizzardFrames(taint, ...)
 
 		if( taint ) then
 			frame.Show = ShadowUF.noop
-		else
+		elseif not( EditModeManagerFrame ) then
```

Comment:
Shouldn't this be EditModeManagerFrame:IsShown() like
Because your current implementation didn't work for me.

```
elseif not( EditModeManagerFrame:IsShown() ) then
```

Reply:
I forgot to update.
It works for me for more than a week.

```suggestion
		if( taint ) then
			frame.Show = ShadowUF.noop
		elseif not( EditModeManagerFrame ) then --Classic?
			frame:SetParent(ShadowUF.hiddenFrame)
			frame:HookScript("OnShow", rehideFrame)
		else
			EditModeManagerFrame:HookScript("OnShow", function(self)
				frame:SetParent(old_parent)
				end
			)
			EditModeManagerFrame:HookScript("OnHide", function(self)
				frame:SetParent(ShadowUF.hiddenFrame)
				rehideFrame(frame)
				end
			)
		end
```

Reply:
This works for a week without errors.

		if( taint ) then
			frame.Show = ShadowUF.noop
		elseif not( EditModeManagerFrame ) then --Classic?
			frame:SetParent(ShadowUF.hiddenFrame)
			frame:HookScript("OnShow", rehideFrame)
		else
			EditModeManagerFrame:HookScript("OnShow", function(self)
				frame:SetParent(old_parent)
				end
			)
			EditModeManagerFrame:HookScript("OnHide", function(self)
				frame:SetParent(ShadowUF.hiddenFrame)
				rehideFrame(frame)
				end
			)
		end